scala_proto/BUILD

```diff
@@ -0,0 +1,39 @@
+load("//scala_proto:scala_proto_toolchain.bzl", "scala_proto_toolchain")
+
+toolchain_type(
+    name = "toolchain_type",
+    visibility = ["//visibility:public"],
+)
+
+scala_proto_toolchain(
+    name = "default_toolchain_impl",
+    with_grpc = False,
+    with_flat_package=False,
+    # with_java=False,
```

Comment:
why the comment? to show the setting?

Reply:
Right now this is commented because i haven't figured out how it should be implemented dynamically -- we need to depend on the java aspect only when this is true. And depending on the java aspect itself since its defined in java i haven't found the right magic invokcation for yet